src/components/shared/Property.tsx

```diff
@@ -13,7 +13,7 @@ export interface IProperty {
 export const Property: React.FunctionComponent<IProperty> = ({ node, onGoToRef }) => {
   const type = isRefNode(node) ? '$ref' : isCombinerNode(node) ? node.combiner : node.type;
   const subtype =
-    type === SchemaKind.Array && (node as IArrayNode).items !== undefined
+    type === SchemaKind.Array && _isObjectLike((node as IArrayNode).items)
```

Comment:
Should we start creating some type guards, like `isArrayNode`, and using those?

Any other spots where we might end up w unexpected null. `properties`, `anyOf`, `allOf`, `oneOf`, etc?

Reply:
> Should we start creating some type guards, like isArrayNode, and using those?

Yeah, sounds indeed a look a good idea. Will do.

> Any other spots where we might end up w unexpected null. properties, anyOf, allOf, oneOf, etc?

Yes there might be other spots, _but_ we take nulls into account in a few places, but just not all.